examples/openvino/image-classification/configs/swin-base-movement-sparsity.json

```diff
@@ -0,0 +1,15 @@
+{
+    "algorithm": "movement_sparsity",
+    "params": {
+        "warmup_start_epoch": 2,
+        "warmup_end_epoch": 5,
+        "importance_regularization_factor": 0.030,
+        "enable_structured_masking": true
+    },
+    "sparse_structure_by_scopes": [
```

Comment:
Why does this only about pruning (not pruning + quantization)? From these two methods, quantization is the simplest one and delivers better performance. Most likely, user will choose quantization from two alternatives.  Stacking it with pruning brings additional benefits.

Reply:
WIP

Reply:
Added [here](https://github.com/vuiseng9/optimum-intel/blob/pr-nncf-jpqd/examples/openvino/question-answering/README.md#joint-pruning-quantization-and-distillation-jpqd-for-bert-on-squad10).